ChangeLog

```diff
@@ -55,6 +55,11 @@
    * tests/test_{counting_single,subset_graph}.py: make explicit the use of
    floor division behavior.
 
+2015-03-07  Alex Hyer  <theonehyer@gmail.com>
```

Comment:
Since your pull request will be the latest addition to the project when we merge it should be at the top of the file with a date newer than the preceding entry.